public/apps/configuration/utils/form-row.tsx

```diff
@@ -13,28 +13,40 @@
  *   permissions and limitations under the License.
  */
 
+import { EuiFormRow, EuiLink, EuiText } from '@elastic/eui';
 import React from 'react';
-import { EuiFormRow, EuiText, EuiTitle } from '@elastic/eui';
```

Comment:
Just curious whether the location change is intentional or have been done by eslint?

Reply:
Done by prettifier